1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -283,16 +283,16 @@ def create_infofiles(row, filenames_file, not_ok_file):
     
     if pd.notnull(row["Personnamn / fotograf"]):
         if "Apenes" in row["Personnamn / fotograf"]:
-            infotext +="|photographer       =  " + "{{creator:Sigvald_Linné}} / Ola Apenes\n"
+            infotext +="|photographer       = " + "{{creator:Sigvald_Linné}} / Ola Apenes\n"
         elif "Sigvald" in row["Personnamn / fotograf"]:
             linne_category == True
-            infotext +="|photographer       =  " + "{{creator:Sigvald_Linné}}\n"
+            infotext +="|photographer       = " + "{{creator:Sigvald_Linné}}\n"
         else:
-            infotext += "|photographer       =  " + row["Personnamn / fotograf"].strip() + "\n"
+            infotext += "|photographer       = " + row["Personnamn / fotograf"].strip() + "\n"
     
     if pd.isnull(row["Personnamn / fotograf"]):
         if row["Personnamn / fotograf"] == "Linné, Sigvald": # not all cases
```

Comment:
Don't think this `if` is needed since it can never be true (since `row["Personnamn / fotograf"]`) is `null` per the `if` before.